financials/Funding request template.txt

```diff
@@ -0,0 +1,24 @@
+## Template for funding request
+This funding request template was made based on past experiences and debates on the Grin Forum about the necessity to have clear, 
+transparent and unambiguous information in funding requests. 
+This template is not intended to be exhaustive and only serves as a guideline to clearly communicate the minimal amount of information 
+needed for the GRIN community to assess requests for funding. Feel free to add and modify to this template to improve clarity and transparency. 
+However, let us aim to keep this funding template minimal and elegant like GRIN and the Mimble Wimble protocol itself. 
+For ease of comparison of funding requests, it is advised to provide these fields of information in the order given below.
+
+## Financial information
+�	Funding period: 

```

Comment:
Looks like something went wrong in the formatting here, I see `�` in the code. In markdown, you can make a bullet by a star and space `* `, and then you can make a sub-bullet by adding three spaces before the bullet, like so:

```
* Bullet 1
   * Sub-bullet 1
   * Sub-bullet 2
* Bullet 2
* Bullet 3
```

Could you update?